recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "moab" %}
-{% set version = "5.4.1rc1" %}
+{% set version = "5.5-RC1" %}
```

Comment:
```suggestion
{% set version = "5.5.0rc11" %}
```
I think we want to be consistent with previous test releases.

Reply:
Obviously, conda did not accept the `-` in the version number in any case.

Reply:
Yeah I didn't realize conda doesn't like the hyphen. I'll have to rename the tarball and make the change here as well.

Reply:
No, you can leave the tarball as it is, no problem.  The URL is hard-coded.

Reply:
That's what we did last time.